.github/workflows/update-cicd-dev-network.yml

```diff
@@ -0,0 +1,87 @@
+name: UPDATE-DEV

```

Comment:
There's a lot of duplication here 
There are 2 workflows that do exactly the same thing with only the network changed - why not have a single workflow that allows you to choose the nwrwork
The second bit of duplication is the deployment itself - there is already a custom action built for deploying 0chain, it does what you're trying to do here except it also tears down existing networks, and there's logic to make sure the deployment was successful - why not use it rather than duplicating the deployment logic in multiple places?